web/wp-content/themes/borealis-2021/tailwind-plugins/container.js

```diff
@@ -46,7 +46,13 @@ module.exports = plugin(function({ addUtilities }) {
         '.container--full': {
             maxWidth: `${breakpoints.xl}px`,
             margin: '0 auto'
-        }
+        },
+        '.border-radius-10': {
+            borderRadius: `10px`,
```

Comment:
So instead of adding these here, you should be able to add them to the Tailwind config file. Details [here](https://tailwindcss.com/docs/border-radius#using-custom-values)